config/filterMap.py

```diff
@@ -0,0 +1,16 @@
+# Mapping of camera filter name: reference catalog filter name; each reference filter must exist in the refcat. 
+# Note that this does not perform any bandpass corrections: it is just a lookup.
+for source, target in [
+        ("u", "g"),
+        ("g", "g"),
+        ('r', 'r'),
+        ('i', 'i'),
+        ('z', 'z'),
+        ('y', 'y'),
```

Comment:
Remove all of the above filterMaps: redundant filtermaps like these are unnecessary and can result in schema errors.